core/src/main/java/io/camunda/migrator/Pagination.java

```diff
@@ -89,8 +95,17 @@ public List<T> toList() {
   public Map<String, Object> toVariableMap() {
     Map<String, Object> result = new HashMap<>();
     toList().forEach(e -> {
-      VariableInstance var = (VariableInstance) e;
-      result.put(var.getName(), var.getValue());
+      VariableInstanceEntity var = (VariableInstanceEntity) e;
+      TypedValue typedValue = var.getTypedValue(false);
+      if (typedValue.getType().equals(ValueType.OBJECT)) {
+        // don't serialize the value for object
+        result.put(var.getName(), typedValue.getValue());
```

Comment:
❓  We don't need to call to `toString` in this case, or is that the default behaviour?